README.md

```diff
@@ -54,6 +54,15 @@ machine will not have all the capabilities of the production system.
 
        make deploy
 
+## Related repositories
+
+- [Web Platform Tests](https://github.com/w3c/web-platform-tests): The tests
```

Comment:
My favorite spelling is web-platform-tests, as in https://github.com/w3c/web-platform-tests/blob/master/README.md